scripts/zones/Chocobo_Circuit/npcs/Alangriche.lua

```diff
@@ -0,0 +1,22 @@
+-----------------------------------
+-- Area: Chocobo_Circuit
+-- NPC: Alangriche
+-- Novice San d'Oria CRA Branch
+-- pos

```

Comment:
All new NPCs will need their `!pos` comments filled

Reply:
I don't know how to do an id shift, this NPC is not loading into the map? Same for the Aht Urhgan Whitegate and [-151.2092 -0.0686 -388.0844] Windurst Woods [-140.2342 -0.0686 -534.2736] sections

There is an area replaced by synthsizers which I also dont know how to correct []